move-vm-backend-common/src/gas_schedule.rs

```diff
@@ -213,22 +213,23 @@ lazy_static! {
                     per_byte_searched: 36.into(),
                 },
             },
+            // vectors will be charged by bytecode operations (see table above).
             vector: move_stdlib::natives::vector::GasParameters {
-                empty: move_stdlib::natives::vector::EmptyGasParameters { base: 1000.into() },
-                length: move_stdlib::natives::vector::LengthGasParameters { base: 1000.into() },
+                empty: move_stdlib::natives::vector::EmptyGasParameters { base: 0.into() },
+                length: move_stdlib::natives::vector::LengthGasParameters { base: 0.into() },
                 push_back: move_stdlib::natives::vector::PushBackGasParameters {
-                    base: 1000.into(),
-                    legacy_per_abstract_memory_unit: 1000.into(),
+                    base: 0.into(),
+                    legacy_per_abstract_memory_unit: 0.into(),
                 },
-                borrow: move_stdlib::natives::vector::BorrowGasParameters { base: 1000.into() },
-                pop_back: move_stdlib::natives::vector::PopBackGasParameters { base: 1000.into() },
-                destroy_empty: move_stdlib::natives::vector::DestroyEmptyGasParameters { base: 1000.into() },
-                swap: move_stdlib::natives::vector::SwapGasParameters { base: 1000.into() },
+                borrow: move_stdlib::natives::vector::BorrowGasParameters { base: 0.into() },
+                pop_back: move_stdlib::natives::vector::PopBackGasParameters { base: 0.into() },
+                destroy_empty: move_stdlib::natives::vector::DestroyEmptyGasParameters { base: 0.into() },
+                swap: move_stdlib::natives::vector::SwapGasParameters { base: 0.into() },
             },
             balance: move_stdlib::natives::balance::GasParameters {
-                transfer: move_stdlib::natives::balance::TransferGasParameters { base: 1000.into() },
-                cheque_amount: move_stdlib::natives::balance::ChequeAmountGasParameters { base: 1000.into() },
-                total_amount: move_stdlib::natives::balance::TotalAmountGasParameters { base: 1000.into() },
+                transfer: move_stdlib::natives::balance::TransferGasParameters { base: 8254.into() },
+                cheque_amount: move_stdlib::natives::balance::ChequeAmountGasParameters { base: 2595.into() },
+                total_amount: move_stdlib::natives::balance::TotalAmountGasParameters { base: 2513.into() },
```

Comment:
Let's increase these by 50% or 100%, whatever you like more, since we need to compensate the missing address conversion functions that were not included in our benchmark.